.github/workflows/codeigniter.yml

```diff
@@ -104,6 +104,13 @@ jobs:
         run: |
           composer update
           composer install --no-progress --prefer-dist --optimize-autoloader
+          sudo apt-get update
+          # html-beautify from the debian package doesn't work for some reason
+          #   sudo apt-get install -y node-js-beautify
+          # Install npm and install js-beautify from there
+          sudo apt-get install -y npm
+          sudo npm update --verbose -g npm
+          sudo npm install --verbose -g js-beautify
```

Comment:
What is the purpose for installing this packages? As far as I understand the `fix_code_style.sh` is not called in the CI pipeline.

Reply:
Installing js-beautify which contains html- beautify which is called by. fix_code_style.sh.

 fix_code_style.sh is called in the yml. In the second job, the one where this installation is done  Check again.

Reply:
Got it. Sorry, I missed that.